charts/team-ns/templates/argocd/argocd-applicationset.yaml

```diff
@@ -80,6 +80,14 @@ spec:
           kind: ValidatingWebhookConfiguration
           jqPathExpressions:
           - '.webhooks[]?.clientConfig.caBundle'
+        - group: admissionregistration.k8s.io
+          kind: MutatingWebhookConfiguration
+          jqPathExpressions:
+          - '.webhooks[]?.clientConfig.caBundle'
+        - group: apps
+          kind: StatefulSet
+          jsonPointers:
+          - /spec/volumeClaimTemplates
```

Comment:
What is the diff that is causing this one? At least of standard resources, ArgoCD should be able to do this properly. Is it a known issue of ArgoCD?

Reply:
I encountered this issue with the Milvus Helm chart: https://github.com/zilliztech/milvus-helm/tree/master/charts/milvus. I don't know if this is a known issue. This is what solved it for this chart.